src/layouts/app/connection-mixin.js

```diff
@@ -155,6 +154,9 @@ export default (superClass) =>
         getState()
       );
 
+      // getActiveTranslation need call callWS method
+      this.hass.language = await getActiveTranslation(this.hass);
```

Comment:
By awaiting this, we're going to block all of startup just to sync local and backend languages, although in most cases they are already in sync.

Reply:
Change to then() and event to un-block the flow